scripts/before_script_postgres.sh

```diff
@@ -7,6 +7,7 @@ echo "[SCRIPT] Before Script :: Setup Parse DB for Postgres"
 PGPASSWORD=postgres psql -v ON_ERROR_STOP=1 -h localhost -U postgres <<-EOSQL
     CREATE DATABASE parse_server_postgres_adapter_test_database;
     \c parse_server_postgres_adapter_test_database;
+    CREATE EXTENSION pgcrypto;
```

Comment:
Is this a breaking change? Will users have to install this EXTENSION in order to use the server? If so can you update the changelog.

Reply:
It's non-breaking, if they were using UUIDs before, they would have already added this. This is only here for us to use the UUID generating function in the test cases (which we weren't using before)

Reply:
Oh this is for the test spec. I thought it was for the Adapter. My bad